### DISCUSSION TITLE
Profiles for Charging and Discharging

### DISCUSSION THREAD

**Discussion by Charliecoop** [JamesCJ60/Universal-x86-Tuning-Utility#94] [Ideas]
I'm looking for a profile that makes the computer go fast as possible (with my negative core offset) and as effecient/slow (within reason!) as possible when I'm on battery. I can't figure out a way to do this at the moment as the automations only work with the 'Custom Presets' and not the 'Premade Presets' which includes that cool ECO mode option.

**Comment by PeterSmoofwah** [comment]
Basically look up your tdp limit for the cpu and set the watts close to that and set the curve optimizer as low as It can go and test for core stretching and stability with OCCT and CineBench for stress testing and stability as you change the curve optimizer. 

Use HwInfo64 to check for core stretching which is noticed by looking at clock speeds vs effective clock speeds. Basically make sure they're similar to each other on all cores. While under load.

It's a long process but simple.
